<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Termometro</class>
 <widget class="QMainWindow" name="Termometro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Termometro</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/imagenes/recursos/icono.png</normaloff>:/imagenes/recursos/icono.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="grpCent">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(175, 214, 255);</string>
        </property>
        <property name="title">
         <string>Cent√≠grados</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLCDNumber" name="outCent">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(240, 178, 255);
background-color: rgb(255, 219, 208);
background-color: rgb(251, 217, 255);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDial" name="cmdCent">
           <property name="minimum">
            <number>-273</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>8</number>
           </property>
           <property name="sliderPosition">
            <number>8</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="grpFahr">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(253, 255, 228)</string>
        </property>
        <property name="title">
         <string>Fahrenheit</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLCDNumber" name="outFahr">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(220, 138, 221);
background-color: rgb(170, 170, 255);
background-color: rgb(201, 213, 255);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDial" name="cmdFahr">
           <property name="minimum">
            <number>-459</number>
           </property>
           <property name="maximum">
            <number>212</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QGroupBox" name="groupKelv">
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="tabletTracking">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(223, 211, 255);
selection-background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 255, 255, 255), stop:0.1 rgba(255, 255, 255, 255), stop:0.2 rgba(255, 176, 176, 167), stop:0.3 rgba(255, 151, 151, 92), stop:0.4 rgba(255, 125, 125, 51), stop:0.5 rgba(255, 76, 76, 205), stop:0.52 rgba(255, 76, 76, 205), stop:0.6 rgba(255, 180, 180, 84), stop:1 rgba(255, 255, 255, 0));</string>
        </property>
        <property name="title">
         <string>Kelvin</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLCDNumber" name="outKelv">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(179, 255, 246);
background-color: rgb(172, 201, 255);
background-color: rgb(255, 224, 210);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDial" name="cmdKelv">
           <property name="minimum">
            <number>273</number>
           </property>
           <property name="maximum">
            <number>373</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>504</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cmdCent</sender>
   <signal>valueChanged(int)</signal>
   <receiver>outCent</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdFahr</sender>
   <signal>valueChanged(int)</signal>
   <receiver>outFahr</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdKelv</sender>
   <signal>valueChanged(int)</signal>
   <receiver>outKelv</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
